gradle/libs.versions.toml

```diff
@@ -1,8 +1,90 @@
 [versions]
+androidx-appcompat = "1.3.0"
+androidx-annotation = "1.5.0"
```

Comment:
Rebase branch onto master and update Annotation dependency. Everything else looks good.